### ISSUE TITLE
Propertygrid under net7 only shows a single item per category

### ISSUE THREAD

**Issue by ykarpeev** [xceedsoftware/wpftoolkit#1731]
After updating my project to net7 I noticed that only a single item per category is shown in category view. It will still show all of them under alphabetical mode.

Here are my results with a test project.

```
        <xctk:PropertyGrid SelectedObject="{Binding}" />


    public class MainViewModel : ObservableObject
    {
        public string Setting2 { get; set; }

        public string Setting1 { get; set; }
    }
```

![image](https://user-images.githubusercontent.com/6986975/202529618-61690060-8dde-4544-99b3-081309f90278.png)

![image](https://user-images.githubusercontent.com/6986975/202529677-951dacd4-1c11-4a16-a141-4016fdf4214b.png)

**Comment by XceedBoucherS** [comment]
Hello,
This is interesting. 

Everything is correct in NET6 but not in NET7.
we will look into this.

Thank you for pointing this out.

**Comment by XceedBoucherS** [comment]
Here's a fix (it will part of v4.6).

In file Xceed.Wpf.Toolkit/PropertyGrid/Implementation/PropertyItemsControl.cs, replace the constructor with: 
public PropertyItemsControl()
    {
      this.Initialized += this.PropertyItemsControl_Initialized;
    }

Then, add the 2 following methods:
private void PropertyItemsControl_Initialized( object sender, EventArgs e )
    {
      var propertyItemsControl = sender as PropertyItemsControl;
      if( propertyItemsControl != null )
      {
        var propertyGrid = propertyItemsControl.TemplatedParent as PropertyGrid;
        if( propertyGrid != null )
        {
          if( propertyGrid.IsVirtualizing )
          {
            this.SetVirtualizingWhenGrouping();
          }
        }
        else
        {
          var propertyItem = propertyItemsControl.TemplatedParent as PropertyItem;
          if( propertyItem != null )
          {
            propertyGrid = propertyItem.ParentElement as PropertyGrid;
            if( propertyGrid != null )
            {
              if( propertyGrid.IsVirtualizing )
              {
                this.SetVirtualizingWhenGrouping();
              }
            }
          }
        }
      }
    }

    private void SetVirtualizingWhenGrouping()
    {
      var propertyItemsControlProperties = TypeDescriptor.GetProperties( this, new Attribute[] { new PropertyFilterAttribute( PropertyFilterOptions.All ) } );
      var prop1 = propertyItemsControlProperties.Find( "VirtualizingPanel.IsVirtualizingWhenGrouping", false );
      if( prop1 != null )
      {
        prop1.SetValue( this, true );
      }
      var prop2 = propertyItemsControlProperties.Find( "VirtualizingPanel.CacheLengthUnit", false );
      if( prop2 != null )
      {
        prop2.SetValue( this, Enum.ToObject( prop2.PropertyType, 1 ) );
      }
    }

**Comment by ykarpeev** [comment]
Thank you.
